<dec f='codebrowser/Include/pyerrors.h' l='92' type='void PyErr_SetExcInfo(PyObject * , PyObject * , PyObject * )'/>
<use f='codebrowser/Modules/_testcapimodule.c' l='2186' u='c' c='test_set_exc_info'/>
<def f='codebrowser/Python/errors.c' l='346' ll='363' type='void PyErr_SetExcInfo(PyObject * p_type, PyObject * p_value, PyObject * p_traceback)'/>
